src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -195,7 +195,16 @@ private Guid GetProjectGuid()
                 return Guid.Empty;
             }
 
-            return vsSolution.GetGuidOfProjectFile(project.FileName);
+            try
+            {
+                return vsSolution.GetGuidOfProjectFile(project.FileName);
+            }
+            catch (Exception ex)
+            {
+                logger.WriteLine(Strings.TextBufferIssueTracker_ProjectGuidError, FilePath, ex);
```

Comment:
This log is a bit spammy for cmake projects; do you think it would be best not to log anything at all?

Reply:
There's no point in logging the call stack; we know where we are in our code, and the call stack inside the VS code isn't useful. I'd just log the message.

We log a message elsewhere saying the file can't be analysed, don't we?

If that's the case, we're not adding any useful to the user here, so I'd log it as a debug message.